distributor/src/distributor/website.bal

```diff
@@ -192,11 +192,59 @@ service mediaWebsite on mediaListener {
             return caller->ok("Still good");
         } else {
             http:Response hr = new;
-            hr.statusCode = 404;
+            hr.statusCode = http:STATUS_NOT_FOUND;
             hr.setTextPayload("This version is no longer active; please upgrade (see status message).");
             return caller->respond(hr);
         }
     }
+
+    @http:ResourceConfig {
+        path: "/sms/{mobileNo}",
+        methods: ["GET"]
```

Comment:
Really should be POST - this is wrong HTTP protocol usage as GET is not supposed to have this kind of side effects.